<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>650</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-image:url(://FONDO.png);
}
#instrucciones{
color:white;
}
#tiempo{
color:white;
}
#puntaje{
color:white;
}
#lblPuntos {
color:white;
qproperty-alignment: AlignCenter;
backgroung: turquoise;
border:3px solid silver;
border-radius:7px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="instrucciones">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Escoge 2 tarjetas e intenta que coincidan</string>
    </property>
   </widget>
   <widget class="QLabel" name="tiempo">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>tiempo</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>721</width>
      <height>551</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Carta01">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta02">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta03">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta04">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta05">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>400</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta06">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>400</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta08">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta09">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta10">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>140</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta11">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>400</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta12">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>270</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta16">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>270</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta14">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>400</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta15">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Carta07">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>134</width>
       <height>129</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="puntaje">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Puntos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPuntos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>430</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>iniciar </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
